src/devtools/views/Profiler/CommitFlamegraph.js

```diff
@@ -123,9 +123,13 @@ function CommitFlamegraph({
   const selectedChartNode = useMemo(() => {
     let chartNode = null;
     if (selectedFiberID !== null) {
-      chartNode = ((chartData.rows[selectedChartNodeIndex].find(
+      const foundChartNode = chartData.rows[selectedChartNodeIndex].find(
         chartNode => chartNode.id === selectedFiberID
-      ): any): ChartNode);
+      );
+
+      if (foundChartNode !== undefined) {
+        chartNode = foundChartNode;
+      }
```

Comment:
Hm, so the problem was that we were assigning an unexpected `undefined` value

Reply:
exactly. the case was that I had a selected a node in one commit, but on subsequent commits the tree had a modification on the parent components (adding another wrapper to the whole tree), so the selected node ID wasn't in the tree anymore.

Reply:
Nice fix